server/master/src/main/java/org/apache/accumulo/master/tableOps/TableRangeOp.java

```diff
@@ -38,19 +37,24 @@
   private static final long serialVersionUID = 1L;
 
   private final String tableId;
+  private final String namespaceId;
```

Comment:
The [Java Object Serialization Specification] seems to indicate that order doesn't matter, also. Very cool.

[1]: https://docs.oracle.com/javase/7/docs/platform/serialization/spec/version.html